gravitee-apim-e2e/ui-test/model/apis.ts

```diff
@@ -17,6 +17,7 @@ import { Endpoint, Services } from './api-endpoints';
 import { Flow, FlowMode } from './api-flows';
 import { MethodType, Plan, Step } from './plan';
 export interface Api {
+  proxy: any;
```

Comment:
Do we have an object type for the proxy ?

Reply:
No. It wiil be removed soon. (after cypress migration)